hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/IncrementalTableBackupClient.java

```diff
@@ -166,6 +167,26 @@ protected List<BulkLoad> handleBulkLoad(List<TableName> tablesToBackup) throws I
       Path tblDir = CommonFSUtils.getTableDir(rootdir, srcTable);
       Path p = new Path(tblDir, regionName + Path.SEPARATOR + fam + Path.SEPARATOR + filename);
 
+      // For continuous backup: bulkload files are copied from backup directory defined by
+      // CONF_CONTINUOUS_BACKUP_WAL_DIR instead of source cluster.
+      String backupRootDir = conf.get(CONF_CONTINUOUS_BACKUP_WAL_DIR);
```

Comment:
[nit] somehow, `hbase.backup.continuous.wal.dir` is become the root now for both WALs and Bulkloads, shouldn't we call it as `hbase.backup.continuous.dir`/`hbase.backup.continuous.root.dir`